server/service/src/item_stats.rs

```diff
@@ -117,9 +131,10 @@ impl ItemStats {
                 item_id: stock_on_hand.item_id.clone(),
                 item_name: stock_on_hand.item_name.clone(),
                 average_monthly_consumption: consumption_map
-                    .get(&stock_on_hand.item_id)
-                    .map(|consumption| *consumption / amc_lookback_months as f64)
-                    .unwrap_or_default(),
+                .get(&stock_on_hand.item_id)
+                .map(|consumption| *consumption / amc_lookback_months)
```

Comment:
ah. gotcha. I was going to ask if we could convert `amc_lookback_months` to `u32` but this is nicer